### DISCUSSION TITLE
Ctrl+character problem in kubuntu and fcitx5

### DISCUSSION THREAD

**Discussion by burbilog** [fcitx/fcitx5#588] [Q&A]
I installed fcitx5 in kubuntu 22.04 (apt install fcitx5 fcitx5-mozc, fcitx is 5.0.14-1 and kubuntu is 22.04.1) and created two groups -- one group is en/ru and second group is en/ja. It works. But if I switch to en in any group, I can't press any ctrl+ combination, for example Ctrl+c. Instead of sending control charactes, fcitx5 sends second layout letters! How to switch to normal mode? I w ork in unix shell and I need control characters like ctrl+c...

Is it possible to turn of this kind of behaviour?

**Comment by wengxt** [comment]
can you share your ~/.config/fcitx5/profile so we can know your setup?

**Reply by wengxt** [reply]
actually, fcitx5-diagnose output would be better. you can use gist to paste it.

**Reply by burbilog** [reply]
Well... i uninstalled/reinstalled it (did not help) and then fiddled with settings, now the problem is gone... and I have no idea what I did.

The trouble it, for some strange reason fcitx5 settings page is a mix of russian and japanese (!), despite english being default language for KDE (just checked, American English is default: https://imgur.com/a/lsM2Ty0 ). But my japanese is not up to the task, so I poked around the settings, may be I did somethings to turn this behaviour off:  https://imgur.com/a/tyEwsf9 https://imgur.com/a/9pN9CvR

fcitx5-diagnose output is https://gist.github.com/burbilog/5aa6edaf0a054842dfb636c39f01b0d7
